src/idm/user.js

```diff
@@ -0,0 +1,122 @@
+import axios from 'axios';
+import { errorHandler } from '../utils';
+import parseUrl from 'url-parse';
+
+const idm = (base, token) => {
+  base = `${base}`;
+  return ({
+    /**
+    * @summary Get the user information
```

Comment:
It's not clear to me what the diff between `agile.idm.user.getuserInfo()` and `agile.idm.user.getuser()` are  there descriptions are very similar: `Get the user information` & `Show information for a particular user`